apps/chat/src/types/themes.ts

```diff
@@ -2,6 +2,7 @@ export interface Theme {
   displayName: string;
   colors: Record<string, string>;
   'app-logo': string;
+  fontFamily: Record<string, string>;
```

Comment:
Please don't forget to update documentation about themes customization

Reply:
Documentation updated.